tests/test_rebar.py

```diff
@@ -45,8 +46,14 @@ class FooListSchema(m.Schema):
     name = m.fields.String(required=True)
 
 
+if MARSHMALLOW_MAJOR_VERSION == 2:
+    _name_field = m.fields.String(load_from='x-name', required=True)
+else:
+    _name_field = m.fields.String(data_key='x-name', required=True)
+
+
 class HeadersSchema(m.Schema):
-    name = m.fields.String(load_from='x-name', required=True)
+    name = _name_field
```

Comment:
since _name_field isn't used elsewhere, you could keep this a 1-liner with something like
```python
name = m.fields.String(required=True, **{('load_from' if MALLOWV2 else 'data_key'): 'x-name'})
```

If this load_from/data_key difference is needed elsewhere maybe `DATA_KEY = 'load_from' if MV2 else 'data_key'` is the right abstraction, so you could do ` m.fields.String(**{DATA_KEY: 'x-name'})` anywhere it's needed.